tensorrt_llm/_torch/models/modeling_utils.py

```
@@ -925,29 +929,59 @@ def run_concurrently(func,
+            # Process completed tasks as they finish
+            for result in futures.as_completed(future_to_result):
+                arg = future_to_result[result]
+                try:
+                    part_weights = result.result()
+                    if reduce_func:
+                        reduce_func(part_weights)
+                    if pbar:
+                        pbar.update(1)
+                except Exception as e:
+                    logger.error(
+                        f"Error executing {func.__name__} with args {arg}: {str(e)}"
```

Ruff (0.15.17)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)